server/src/test/java/org/spine3/server/stand/StandShould.java

```diff
@@ -561,7 +562,47 @@ public void onNext(QueryResponse value) {
     @Test
     public void retrieve_whole_entity_if_nothing_is_requested() {
         //noinspection ZeroLengthArrayAllocation
-        requestSampleCustomer(new int[] {}, getDuplicateCostumerStreamObserver());
+        requestSampleCustomer(new int[]{}, getDuplicateCostumerStreamObserver());
+    }
+
+    @SuppressWarnings("MethodWithMultipleLoops")
+    @Test
+    public void select_entity_singleton_by_id_and_apply_field_masks() {
+        final Stand stand = prepareStandWithAggregateRepo(InMemoryStandStorage.newBuilder()
+                                                                              .build());
+        final String customerFqn = Customer.getDescriptor()
```

Comment:
Could you please either rename it to `customerDescriptor` or add a line comment such as `//Fully qualified name for {@code Customer} message`.